workflow/workflow-demos/processes/BusinessNotification/OrderProcess.p.json

```diff
@@ -0,0 +1,385 @@
+{
+  "$schema" : "https://json-schema.axonivy.com/process/11.2.2/process.json",
+  "id" : "18DD058BCE48D418",
+  "config" : {
+    "data" : "workflow.order.OrderData"
+  },
+  "elements" : [ {
+      "id" : "f0",
+      "type" : "RequestStart",
+      "name" : "startOrder",
+      "config" : {
+        "signature" : "startOrder",
+        "request" : {
+          "name" : "8.1 Business Notification",
+          "category" : "notification",
+          "customFields" : [
+            { "name" : "cssIcon", "value" : "si si-messages-bubble" }
+          ]
+        },
+        "permission" : {
+          "anonymous" : false
+        },
+        "persistOnStart" : true,
+        "task" : {
+          "name" : "New Order"
+        },
+        "case" : {
+          "name" : "New Order"
+        }
+      },
+      "visual" : {
+        "at" : { "x" : 88, "y" : 96 }
+      },
+      "connect" : [
+        { "id" : "f2", "to" : "f3" }
+      ]
+    }, {
+      "id" : "f1",
+      "type" : "TaskEnd",
+      "visual" : {
+        "at" : { "x" : 1408, "y" : 600 }
+      }
+    }, {
+      "id" : "f3",
+      "type" : "DialogCall",
+      "name" : "Input Order",
+      "config" : {
+        "dialog" : "workflow.order.InputOrder:start()",

```

Comment:
the initial order form is empty ... which is realistic, but bad for demonstrations. 
I'd prefil the order form with 'John Doe' and the Ferrari you are also using in tests.
So I can demonstrate the feature without touching the keyboard...